packages/frontend/src/help/theory_documentation/theory_file_parser.tsx

```diff
@@ -0,0 +1,46 @@
+import { readdirSync, statSync, writeFileSync } from "node:fs";
+import { join, resolve } from "node:path";
```

Comment:
This will not work because you're using Node-specific APIs, which are not available in the browser. That's why the CI is failing. We can discuss alternative approaches.